### DISCUSSION TITLE
Apparently cannot locate wallpapers inside steamapps folder

### DISCUSSION THREAD

**Discussion by MBlan18** [catsout/wallpaper-engine-kde-plugin#270] [General]
I have a dual monitor set up, and I got the wallpapers to show up on one of the monitors, however, linking the same library on the other monitor comes up with absolutely nothing.
I'm running Manjaro with XFCE.

**Comment by Jelgnum** [comment]
I'm impressed you got it to work on xfce, this is a kde plugin

**Comment by MBlan18** [comment]
HA! In that case, I'm surprised I got it to work as well! I kept dictating
what folder it should pull from, and it worked, but only for one monitor.
